sfa_dash/blueprints/util.py

```diff
@@ -299,14 +331,22 @@ def handle_response(request_object):
                     f' <a href="{previous_page}">Return to the previous '
                     'page.</a>')
             raise DataRequestException(request_object.status_code, **errors)
-        # Other errors should be due to bugs and not by attempts to reach
-        # inaccessible data. Allow exceptions to be raised
-        # so that they can be reported to Sentry.
+        elif request_object.status_code == 422:
+            errors = {'422': ['Failed to compute aggregate Values']}
+            raise DataRequestException(request_object.status_code, **errors)
```

Comment:
could remove this copy pasta by setting ``errors = {}`` above, removing the else and then have 
```python
if errors:
    raise DataRequestException(request_boject.status_code, **errors)
else:
    request_object.raise_for_status()
```